lib/modules/metadata.mjs

```diff
@@ -174,6 +174,85 @@ export async function validateObsoleteUpdateRef (doc, { mode = MODES.NORMAL, off
           }
         ))
       })
+
+      if (!offline) {
+        // -> Obsoletes an already obsoleted rfc
+        if (mode !== MODES.SUBMISSION && obsoletesRfc.length > 0) {
+          for (const ref of obsoletesRfc) {
+            if (RFC_NUM_RE.test(ref)) {
+              const rfcInfo = await fetchRemoteRfcInfo(ref)
+              if (!rfcInfo) {
+                result.push(
+                  new ValidationWarning(
+                    'OBSOLETES_RFC_NOT_FOUND',
+                    `The <rfc> field states that it obsoletes RFC ${ref} but no matching RFC could be found on rfc-editor.org.`,
+                    {
+                      ref: 'https://authors.ietf.org/en/rfcxml-vocabulary#obsoletes'
+                    }
+                  )
+                )
+              } else if (rfcInfo.obsoleted_by?.length > 0) {
+                result.push(
+                  new ValidationWarning(
+                    'OBSOLETES_OSOLETED_RFC',
+                    `The <rfc> field states that it obsoletes RFC ${ref} but it's already obsoleted by RFC ${rfcInfo.obsoleted_by.join(
+                      ', '
+                    )}.`,
+                    {
+                      ref: 'https://authors.ietf.org/en/rfcxml-vocabulary#obsoletes'
+                    }
+                  )
+                )
+              }
+            }
+          }
+        }
+
+        // -> Updates an already obsoleted rfc
+        if (mode !== MODES.SUBMISSION && updatesRfc.length > 0) {
+          for (const ref of updatesRfc) {
+            if (RFC_NUM_RE.test(ref)) {
+              const rfcInfo = await fetchRemoteRfcInfo(ref)
+              if (!rfcInfo) {
+                result.push(
+                  new ValidationWarning(
+                    'UPDATES_RFC_NOT_FOUND',
+                    `The <rfc> field states that it updates RFC ${ref} but no matching RFC could be found on rfc-editor.org.`,
+                    {
+                      ref: 'https://authors.ietf.org/en/rfcxml-vocabulary#updates'
+                    }
+                  )
+                )
+              } else if (rfcInfo.obsoleted_by?.length > 0) {
+                result.push(
+                  new ValidationWarning(
+                    'UPDATES_OSOLETED_RFC',
+                    `The <rfc> field states that it updates RFC ${ref} but it's already obsoleted by RFC ${rfcInfo.obsoleted_by.join(
+                      ', '
+                    )}.`,
+                    {
+                      ref: 'https://authors.ietf.org/en/rfcxml-vocabulary#updates'
+                    }
+                  )
+                )
+              } else if (rfcInfo.updated_by?.length > 0) {
+                result.push(
+                  new ValidationWarning(
+                    'UPDATES_UPDATED_RFC',
+                    `The <rfc> field states that it updates RFC ${ref} but it's already updated by RFC ${rfcInfo.updated_by.join(
```

Comment:
This isn't something to warn about - this happens all the time. See how many RFCs have updated RFC3261 for example.